authors/apps/authentication/urls.py

```diff
@@ -6,6 +6,6 @@
 
 urlpatterns = [
```

Comment:
Its better to provide names for all the urls than just one

Reply:
@Sulaiman-Mozes kindly color more on this